roles/common/tasks/system-tools.yml

```diff
@@ -29,6 +29,22 @@
     - cdpr
     - sysstat
     - logrotate
+    - ipmitool
+    - ntp
+    - vlan
+
+- name: load vlan module
+  lineinfile: dest=/etc/modules
+              line="8021q"
+  when: ansible_distribution_version == "14.04"
+
+# http://www.supermicro.com/support/faqs/faq.cfm?faq=14537
+- name: blacklist mei
+  lineinfile: dest=/etc/modprobe.d/blacklist.conf
+              line="blacklist {{ item }}"
+  with_items:
+    - mei
+    - mei_me
```

Comment:
I was actually seeing the errors described by this on the Dell gear with trusty.